src/api-docs/product.docs.js

```diff
@@ -479,57 +479,105 @@ const retrieveProductItems = {
   }
 };
 
-const disableProduct = {
-  tags: ["Product"],
-  description: "make product unavailable",
-  parameters: [
-    {
-      name: "searchParam",
-      in: "query",
-      description: "the text you want to search",
-      type: "string",
-      example: "1"
-    }
-  ],
-  responses: {
-    200: {
-      description: "OK",
-      content: {
-        "application/json": {
-          schema: {
-            type: "object"
+
+   
+  const listProduct = {
+    tags: ["Product"],
+    description: "list all product from seller collection",
+    security: [
+      {
+        token: [],
+      },
+    ],
+    parameters: [],
+    responses: {
+      200: {
+        description: "ok",
+        content: {
+          "application/json": {
+            schema: {
+              type: "object"
+            }
+          }
+        }
+      },
+      404: {
+        description: "Not found",
+        content: {
+          "application/json": {
+            schema: {
+              type: "object"
+            }
           }
         }
       }
     }
-  }
-};
+  };
 
-const enableProduct = {
-  tags: ["Product"],
-  description: "make product available",
-  parameters: [
-    {
-      name: "searchParam",
-      in: "query",
-      description: "the text you want to search",
-      type: "string",
-      example: "2"
+  const enableProduct = {
+    tags: ["Product"],
+    description: "make product available",
+    security: [
+      {
+        token: [],
+      },
+    ],
+    parameters: [
+      {
+        name: "searchParam",
+        in: "query",
+        description: "the text you want to search",
+        type: "string",
+        example: "2"
+      }
+    ],
+    responses: {
+      200: {
+        description: "OK",
+        content: {
+          "application/json": {
+            schema: {
+              type: "object"
+            }
+          }
+        }
+      }
```

Comment:
Same as here; As you did on `clearCart`, I would suggest using the declared responses object to simplify the code and reduce duplication. If you do that, we'll only have one line here.